Author: Adam Gandelman <adamg@canonical.com>
Date: Wed Mar  7 10:40:31 PST 2012
Subject: Temporarily patch setup.py

Namespace issue was overlooked in recent upstream commit.  Patch locally
temporarily

Index: nova-2012.1/setup.py
===================================================================
--- nova-2012.1.orig/setup.py	2012-03-07 10:07:37.000000000 -0800
+++ nova-2012.1/setup.py	2012-03-07 10:38:44.284260509 -0800
@@ -33,7 +33,7 @@
             for builder in ['html', 'man']:
                 self.builder = builder
                 self.finalize_options()
-                BuildDoc.run(self)
+                setup_command.BuildDoc.run(self)
     nova_cmdclass['build_sphinx'] = local_BuildDoc
 
 except Exception:
